cmd/opm/alpha/render-graph/cmd.go

```diff
@@ -61,6 +62,9 @@ $ opm alpha render-graph quay.io/operatorhubio/catalog:latest | \
 			render.AllowedRefMask = action.RefDCImage | action.RefDCDir | action.RefSqliteImage | action.RefSqliteFile
 			render.Registry = registry
 
+			// Run all migrations
+			render.MigrationLevel = ""
```

Comment:
Doesn't this mean "run no migrations"?

Reply:
Good catch.  I didn't write that, and `opm render-graph` does not care about bundle metadata, so we shouldn't set anything.